test/commonTaskInputs.test.ts

```diff
@@ -118,7 +118,7 @@ describe("getCredentials tests", () => {
     { endpoint: 'https://ppdevtools.crm.dynamics.com', cloudInstance: 'Public' },
     { endpoint: 'https://aurora.crm10.dynamics.com', cloudInstance: 'Tip1' },
```

Comment:
Hmm.  I would expect this test to be failing, as the `Tip1` test here should be resolving to `Preprod` with previous changes in getCredentials.

Reply:
weird, may be a test authoring error? Maybe this test isn't actually getting executed?
not sure how to run the tests locally.
in any case, we should be removing this logic in a follow-up work item. But the wrapper code will need to allow the --cloud argument to be empty/not specified.